frontend/src/hooks/pims-api/useApiDocuments.ts

```diff
@@ -0,0 +1,20 @@
+import { Api_Document_Type } from 'models/api/Document';
+import React from 'react';
+
+import { useAxiosApi } from '.';
+
+/**
+ * PIMS API wrapper to centralize all AJAX requests to the note endpoints.
+ * @returns Object containing functions to make requests to the PIMS API.
+ */
+
+export const useApiDocuments = () => {
+  const api = useAxiosApi();
+
+  return React.useMemo(
+    () => ({
+      getDocumentTypes: () => api.get<Api_Document_Type[]>(`/documents/document-types`),
```

Comment:
nit: not sure we need this.

Reply:
It will needed because of this https://github.com/bcgov/PSP/pull/1760#discussion_r925116695